Makefile

```diff
@@ -8,7 +8,7 @@ DOCNAME = CAOM
 DOCVERSION = 2.5
 
 # Publication date, ISO format; update manually for "releases"
-DOCDATE = 2024-08-21
+DOCDATE = 2024-09-27
```

Comment:
Why this date?

Reply:
ah, that was the publication date of the IVOA WD; I just hadn't merged that back into main so it was still sitting in my fork

Reply:
I didn't update because I'm not publishing a new WD